docs/design/resolve-divergence.md

```diff
@@ -0,0 +1,104 @@
+# Resolve-Divergence Command Design
```

Comment:
nit: I think this can have a better title. Also this is missing an entry in `mkdocs.yml`.